### ISSUE TITLE
UI Accessibility - focus indicators, form issues

### ISSUE THREAD

**Issue by Worfle** [solo-io/gloo-portal-issues#198]
From #197, but also in #196 ::
- Missing required indicators on form
  - Users reported this on the Client Credentials form

**Comment by Worfle** [comment]
Stepping back as we may want a design first for showing focus.

**Comment by Sodman** [comment]
Hey @Worfle  I see this is in done and the linked PR is merged, are we good to close this out?
